test/public/test/call-remote-callbacks.js

```diff
@@ -275,6 +275,29 @@ asyncTest('required radio should only require one to be checked', 1, function()
   }, 13);
 });
 
+asyncTest('required radio should only require one to be checked if not all radios are required', 1, function() {
+  $(document).bind('iframe:loading', function() {
```

Comment:
Not that I particularly care whether `bind` is used vs `on` or whatever, but just wanted to point out that neither `bind` nor `delegate` are deprecated.

You can see where we've discussed the confusion in the jQuery docs in [this forum](https://forum.jquery.com/topic/delegate-vs-on) and Dave Methvin pointed out that they're not deprecated nor are there any plans to remove them (granted, this was 4 years ago).

I've personally always preferred `bind` and `delegate` over `on`, because I dislike how `on` does _completely_ different things depending on the number and order of arguments passed to it, which results in less readable code.

Anyway, again, I don't really care, as `on` seems to have won out. Just wanted to point out that you probably shouldn't replace them due to being deprecated, since they're not. If on the other hand, `on` is simply preferred for those maintaining and using jquery-ujs, that's a much better reason to replace them.